lib/pdc/resource/associations/has_many.rb

```diff
@@ -0,0 +1,23 @@
+require 'pdc/resource/associations/association'
+
+module PDC::Resource
+  module Associations
+    class HasMany < Association
+      def initialize(*args)
+        super
+        @options.reverse_merge!(uri: "#{parent_path}/:#{foreign_key}/#{@name}/(:#{primary_key})")
+        @params[foreign_key] = parent.id
+      end
+
+      def load
+        self
```

Comment:
Maybe we don't need to set the "uri" in association, because in each classes it can parse the uri for us.
Here we just need to add the below line to instead of "self" I think.
`        klass.where(@options[:k_foreign_key] => @params[foreign_key])
`
And also when we create a association we need to add a var like k_foreign_key there.
```
Class Release
   has_many :release_variants, k_foreign_key: :release
end
```

Reply:
I think the uri is used for overriding. The default one is as the above one uri: "#{parent_path}/:#{foreign_key}/#{@name}/(:#{primary_key})" and u can customize it if your request uri is not following the normal one.